knowledge-base/hide-column-headers-maui-datagrid.md

```diff
@@ -0,0 +1,82 @@
+---
+title: How to Hide Column Headers in a DataGrid for MAUI
+description: Learn how to hide the column headers in a DataGrid for .NET MAUI.
+type: how-to
+page_title: Hide Column Headers in DataGrid for MAUI
+slug: hide-column-headers-maui-datagrid
+tags: maui, datagrid, column-headers, hide
+res_type: kb
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 6.7.0 | Telerik UI for .NET MAUI DataGrid | [Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova)| 
+
+## Description
+
+I want to hide the column headers in a DataGrid for MAUI. Is there a way to achieve this?
```

Comment:
Personally, I think this should probably be documented not as a KB but as part of the docs [here](https://docs.telerik.com/devtools/maui/controls/datagrid/columns/header).